src/sentry/utils/email.py

```diff
@@ -35,7 +35,7 @@
 from sentry.utils.strings import is_valid_dot_atom
 from sentry.web.helpers import render_to_string
 
-logger = logging.getLogger(__name__)
+logger = get_logger()
```

Comment:
I'd really prefer defining the name in the actual logging message, but for things like utils we can probably get away with defining it at the module level.